packages/compile-solidity/run.js

```diff
@@ -10,21 +9,27 @@ async function run(rawSources, options) {
     return {
```

Comment:
So perhaps in this case we could consider returning `null` and deal with the return in `index.js`.